modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -567,6 +572,7 @@ class EditorCommands {
 
     self.addQueryValueHandler('FontName', () => FontCommands.fontNameQuery(editor), this);
     self.addQueryValueHandler('FontSize', () => FontCommands.fontSizeQuery(editor), this);
+    self.addQueryValueHandler('LineHeight', () => LineHeightCommands.lineHeightQuery(editor), this);
```

Comment:
Is this query command used anywhere, is it required? There isn't a lot of these query commands since it's usually a pretty slow way of checking that the editor state is.

Reply:
Noticed that it was only used in the tests. Exposing public api for test purposes might not be what we want.

Reply:
It's used in silver to determine which item to check in the line height menu / toolbar.